apps/server/src/shared/infra/calendar/service/calendar.service.ts

```diff
@@ -35,7 +35,7 @@ export class CalendarService {
 		)
 			.then((resp: AxiosResponse<ICalendarEvent>) => this.calendarMapper.mapToDto(resp.data))
 			.catch((error) => {
-				throw new InternalServerErrorException(error);
+				throw error;
```

Comment:
Same InternalServerErrorException was correct. After the change the catch code do nothing.

Reply:
After some thinking, it make sense to remove the throw completly. This handling should be solve with BC-2432.